src/tests/ftest/rebuild/container_rf.yaml

```diff
@@ -15,9 +15,8 @@ server_config:
           scm_mount: /mnt/daos
   system_ram_reserved: 1
 pool:
-  scm_size: 1073741824
-  svcn: 3
-  control_method: dmg
+  size: 1G
+  svcn: 7  # To match number of servers
```

Comment:
The logic should be: "This test kills 3 engines." => "It needs at least 3 * 2 + 1 = 7 PS replicas to avoid losing the PS." => "It needs at least 7 engines for 7 PS replicas."

Reply:
Right. This test has some other issues I need to fix as part of https://daosio.atlassian.net/browse/DAOS-15074.
I want to make this more flexible then

Reply:
@daltonbohning, sorry for the ambiguity. I mean the comment "to match number of servers" should be something along the line of "to allow killing 3 engines".

Reply:
No worries, I understand! I also set the svcn to exactly the number of servers so all ranks will be svc ranks, making this test a little more deterministic until I fix it properly. E.g. there are some cases where I think the test needs to look at which ranks are svc ranks before killing.

Reply:
Good! I think "7 engines and 7 PS replicas" is good enough, for there's no external interface for knowing the current, exact set of PS replicas.